src/AzureMcp.csproj

```diff
@@ -118,6 +143,47 @@
     </ItemGroup>
   </Target>
 
+  <Target
+    Name="UpdateServerJsonForPackaging"
+    DependsOnTargets="_InitializeNuspecRepositoryInformationProperties"
```

Comment:
nit: ideally we shouldn't be relying on 'private' extensibility points like this - is there a publically-documented Pack-related Target that would serve just as well?